README.md

```diff
@@ -129,6 +129,8 @@
 
 [Amazon](https://aws.amazon.com/message/65648/). Human error during a routine networking upgrade led to a resource crunch, exacerbated by software bugs, that ultimately resulted in an outage across all US East Availability Zones as well as a loss of 0.07% of volumes.
 
+[Amazon](https://aws.amazon.com/message/680342/). Inability to contact a data collection server triggered a latent memory leak bug in the reporting agent on the storage servers. And there is no graceful degradation handling, thus reporting agent continuously contacted the collection serveer in a way that slowly consumed system memory. Also monitoring system failed to alarm on this EBS server's memory leak, also EBS servers generally make very dynamic usae of all memory. By Monday morning, the rate of memory loss became quite high and confusmed enough memory on the affected storage servers which caannot keep with the request handling process. This error got further severed by the inability to do the failover, which resulted the outage.
```

Comment:
There are few spelling mistakes.
>  And there is no graceful degradation handling, thus reporting agent continuously contacted the collection **serveer** in a way that slowly consumed system memory. Also monitoring system failed to alarm on this EBS server's memory leak, also EBS servers generally make very dynamic **usae** of all memory. By Monday morning, the rate of memory loss became quite high and **confusmed** enough memory on the affected storage servers which **caannot** keep with the request handling process.

Reply:
Thanks, just fixed the typos and updated the wording.